tensorflow/compiler/mlir/tensorflow/transforms/lower_tf.td

```diff
@@ -195,8 +195,7 @@ def : Pat<(TF_PadOp TensorOf<[AnySignlessInteger, AnyFloat]>:$input, $paddings),
 // Reciprocal op patterns.
 //===----------------------------------------------------------------------===//
 
-// TODO(hinsu): Support complex and unsigned input types.
```

Comment:
reciprocal doesn't take unsigned types on the TF documentation so I removed the comment but let me know if this refers to something else